src/frontends/tensorflow/src/input_model.hpp

```diff
@@ -4,8 +4,8 @@
 
 #pragma once
 
-#include "input_model.hpp"
 #include "openvino/frontend/extension/telemetry.hpp"
+#include "openvino/frontend/input_model.hpp"
 #include "openvino/frontend/tensorflow/graph_iterator.hpp"
 #include "place.hpp"
```

Comment:
nit: do the same for `place.hpp`

Reply:
This is include of internal header of TF FE so no need to make the full path.